src/main/java/film/springbootapplication/controller/GenreController.java

```diff
@@ -4,31 +4,35 @@
 import film.springbootapplication.model.Genre;
 import film.springbootapplication.service.GenreService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Controller;
+import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 import javax.persistence.EntityNotFoundException;
 import java.util.List;
 
 @RestController
-@RequestMapping( value = "/genres")
+@RequestMapping(value = "/genres")
```

Comment:
we don't need @RequestMapping annotation if we already defined this class as @RestController. We can define mapping directly as @RestController annotation parameter